src/App.tsx

```diff
@@ -4,11 +4,20 @@ import { ErrorBoundary } from "react-error-boundary";
 
 import store from "@/stores/redux/store";
 import { errorFallbackRender } from "@/components/error-fallback-render/error-fallback-render";
+import { EmotionCacheProvider } from "@/components/emotion-cache-provider/emotion-cache-provider";
+import { ThemeProvider } from "@emotion/react";
+import { generateTheme } from "@/theme/theme";
 
 export const App = ({ children }: PropsWithChildren): JSX.Element => {
   return (
     <ErrorBoundary fallbackRender={errorFallbackRender}>
-      <Provider store={store}>{children}</Provider>
+      <EmotionCacheProvider>
```

Comment:
please take a look at Prettier VSCode configuration video, if it would be not enough you can try to find more info on how to configure VSCode and Prettier together

https://www.youtube.com/watch?v=__eiQumLOEo

Reply:
resolved